client/keys/show.go

```diff
@@ -106,14 +106,16 @@ func runShowCmd(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	output, _ := cmd.Flags().GetString(cli.OutputFlag)
-
 	switch {
 	case isShowAddr:
-		printKeyAddress(cmd.OutOrStdout(), info, bechKeyOut)
+		err = printKeyAddress(cmd.OutOrStdout(), info, bechKeyOut)
 	case isShowPubKey:
-		printPubKey(cmd.OutOrStdout(), info, bechKeyOut)
+		err = printPubKey(cmd.OutOrStdout(), info, bechKeyOut)
 	default:
-		printKeyInfo(cmd.OutOrStdout(), info, bechKeyOut, output)
+		err = printKeyInfo(cmd.OutOrStdout(), info, bechKeyOut, output)
```

Comment:
LGTM is showing a warning. And for once it's right. Please remove. `golangci-lint` warnings *don't take precedence over  standard, idiomatic Go*.